R/SearchSamples.R

```diff
@@ -19,11 +19,162 @@
 #' @import dplyr
 #' @import RSQLite
 #' @import purrr
-#' @export
+#' @export SearchByType SearchSamples SearchControls
+
+
+SearchByType <- function(sample_storage_type=NULL, control_type=NULL, filters = NULL, format = NULL, database = Sys.getenv("SDB_PATH"), config_yml = Sys.getenv("SDB_CONFIG"), include_internal_sample_id = FALSE) {
```

Comment:
Is there a reason we're using a flagged function like this instead of just calling `searchControls` or `searchSamples` directly?